modules/nf-core/genmap/map/tests/main.nf.test

```diff
@@ -0,0 +1,82 @@
+nextflow_process {
+
+    name "Test Process GENMAP_MAP"
+    config "./nextflow.config"
+    script "../main.nf"
+    process "GENMAP_MAP"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "genmap"
+    tag "genmap/index"
+    tag "genmap/map"
+
+    setup {
+        run("GENMAP_INDEX") {
+            script "../../index/main.nf"
+
+            process {
+                """
+                input[0] = [
+                    [id:"test"],
+                    file(params.modules_testdata_base_path + 'genomics/sarscov2/genome/genome.fasta', checkIfExists:true)
+                ]
+                """
+            }
+        }
+    }
+
+    test("sarscov2 - index bed") {
+
+        when {
+            params {
+                module_prefix = "test_map"
```

Comment:
Could you please just change the default prefix in the module to be `${meta.id}_map`, so that this isn't necessary to use the module but it can still be customised as people want.

Reply:
Sure !